docs/basic-features/data-fetching/get-server-side-props.md

```diff
@@ -74,6 +74,10 @@ export async function getServerSideProps() {
 export default Page
 ```
 
+## Does getServerSideProps render an error page
+
+If an error is thrown inside `getServerSideProps`, it will show an error page when using `next start`. To display an error page, you must create the `pages/500.js` file. Check out the documentation for [500 page](/docs/advanced-features/custom-error-page#500-page) to learn more on how to create it.
```

Comment:
```suggestion
If an error is thrown inside `getServerSideProps`, it will show the `pages/500.js` file. Check out the documentation for [500 page](/docs/advanced-features/custom-error-page#500-page) to learn more on how to create it. During development this file will not be used and the dev overlay will be shown instead. 
```